docs/user-guide/quick-start/contributor-gui.md

```diff
@@ -210,15 +295,22 @@ more time on data analysis. Follow these steps to integrate and curate your data
 ### Access the Samples Tab:
 
 * Click on the **Samples** tab from the main study screen to explore previously uploaded data. 
-* To start the curation process, click on **Edit** at the bottom left corner of your window.
+* To start the curation process, click on **Edit** in the bottom left corner of your window.
 
+<figure markdown="span">
 ![Sample Edit](quick-start-images/samples-edit.png)
+<figcaption>To curate data, navigate to the <strong>Samples</strong> tab (where the samples metadata is imported) and click on the Edit button at the bottom of the screen</figcaption>
+</figure>
 
 ### Identify any data
 
 * Identify any data that is not valid according to the applied template. 
 Invalid data will be highlighted in **red** under the yellow template columns.
-  ![Select invalid value](quick-start-images/invalid-data.png)
+
+<figure markdown="span">
+![Select invalid value](quick-start-images/invalid-data.png)
+<figcaption>Invalid data that does not follow the template rules will be highlighted in <strong>red</strong>, while valid data will be shown in <strong>green</strong></figcaption>
```

Comment:
>while valid data will be shown in green

it's not fully true, because we highlight in green only data in columns that a) have type = text and b) have connected dictionary and c) value is taken from the dictionary = value is recognised as a valid value, for all other columns/types/cases valid data won't be highlighted in green